frontend/src/shared-components/donation-details/donation.details.jsx

```diff
@@ -2,46 +2,43 @@ import React from "react";
+      <span className={styles.paymentSlipItemValue}>{value || ""}</span>
+    </div>
+  );
 }
 
 function FinancialInfo({ financialInfo }) {
-
-	return (
-		<div className={styles.rightSide}>
-			<h3 className={styles.paymentSlipTitle}>Informacije za uplatu</h3>
-			<PaymentSlipItem label="Svrha uplate:" value={financialInfo.payment_purpose}/>
-			<PaymentSlipItem label="Primalac:" value={financialInfo.payment_receiver}/>
-			<PaymentSlipItem label="Šifra plaćanja:" value={financialInfo.payment_code}/>
-			<PaymentSlipItem label="Račun primaoca:" value={financialInfo.payment_bank_account}/>
-			<PaymentSlipItem label="Model:" value={financialInfo.payment_model}/>
-			<PaymentSlipItem label="Poziv na broj:" value={financialInfo.payment_reference_number}/>
-		</div>
-	);
+  return (
+    <div className={styles.rightSide}>
+      <h3 className={styles.paymentSlipTitle}>Informacije za uplatu</h3>
+      <PaymentSlipItem label="Svrha uplate:" value={financialInfo.payment_purpose} />
+      <PaymentSlipItem label="Primalac:" value={financialInfo.payment_receiver} />
+      <PaymentSlipItem label="Šifra plaćanja:" value={financialInfo.payment_code} />
+      <PaymentSlipItem label="Račun primaoca:" value={financialInfo.payment_bank_account} />
+      <PaymentSlipItem label="Model:" value={financialInfo.payment_model} />
+      <PaymentSlipItem label="Poziv na broj:" value={financialInfo.payment_reference_number} />
+    </div>
+  );
 }
 
 export const DonationDetails = ({ donation }) => {
-	const shouldShowFinancialInfo = donation.is_active && donation.financial_info
+  const shouldShowFinancialInfo = donation.is_active && donation.financial_info;
 
-	return (
-		<>
-			<h2 className={styles.h2}>{donation.subtitle}</h2>
-			<div className={styles.donationDetails}>
-				<div className={shouldShowFinancialInfo ? styles.leftSide: styles.leftSideOnly}>
-					<h3 className={styles.leftSideTitle}>Opšte informacije</h3>
-					<div className={styles.donationTypeTag}>
-						{donation.financial_info ? "Finansijska pomoć" : "Robna pomoć"}
-					</div>
-					<p className={styles.p1}>{donation.description}</p>
-				</div>
-				{shouldShowFinancialInfo && <FinancialInfo financialInfo={donation.financial_info}/>}
-			</div>
-			{/*	TODO: ADD Payment Slip LINK */}
-		</>
-	);
+  return (
+    <>
+      <h2 className={styles.h2}>{donation.subtitle}</h2>
+      <div className={styles.donationDetails}>
+        <div className={shouldShowFinancialInfo ? styles.leftSide : styles.leftSideOnly}>
+          <h3 className={styles.leftSideTitle}>Opšte informacije</h3>
+          <div className={styles.donationTypeTag}>{donation.financial_info ? "Finansijska pomoć" : "Robna pomoć"}</div>
+          <p className={styles.p1}>{donation.description}</p>
+        </div>
+        {shouldShowFinancialInfo && <FinancialInfo financialInfo={donation.financial_info} />}
+      </div>
+      {/*	TODO: ADD Payment Slip LINK */}
+    </>
+  );
 };

```

Comment:
Isti komentar kao i za prethodni fajl.
Najbolje bi bilo da se ozve izmene revert-uju za svaki slucaj.

Reply:
@milos1302 Testirao sam i sve radi kako treba, nisam siguran zasto je gh zabelezio to kao promenu, ali tako se ponasa sa bilo kojim fajlom u koji udjem.